src/components/editor/JSONEditor.tsx

```diff
@@ -1,16 +1,30 @@
 import 'codemirror/mode/javascript/javascript';
+

```

Comment:
I thought the line break was to separate the import from different modules and our relative imports

Reply:
VSCode also does that. I guess a general import like that is another category.